src/ButtonLink/helpers/getTypeToken.js

```diff
@@ -28,6 +30,10 @@ const getTypeToken: GetTypeToken = name => ({ theme, type }) => {
       [TYPES.PRIMARY]: theme.orbit.colorTextButtonLinkPrimaryActive,
       [TYPES.SECONDARY]: theme.orbit.colorTextButtonLinkSecondaryActive,
     },
+    [TOKENS.backgroundButtonFocus]: {
+      [TYPES.PRIMARY]: "rgba(0, 169, 145, 0.1)", // TODO: Fix convert HexToRgba and tokenize
```

Comment:
Can you create issue on `https://github.com/kiwicom/orbit-design-tokens/` 🙏

Reply:
Sure